src/components/searchBar/SearchBar.jsx

```diff
@@ -1,14 +1,18 @@
 import Axios from 'axios';
 import React, { useState } from 'react';
+import Eye from '../eye/Eye';
+import FilterButton from '../filterButton/FilterButton';
+
 import ResultCard from '../ResultCard/ResultCard';
 import './Search.css';
 
-const Search = () => {
+const SearchBar = () => {
   const [query, setQuery] = useState('');
   const [results, setResults] = useState([]);
 
   const onChange = (e) => {
     e.preventDefault();
+
```

Comment:
Useless empty line
+ the previous line is useless too: a "change" event has no default behaviour to cancel :shrug: